assets/prototype/domain/shared/predicates/dateUtilities/sortDates.ts

```diff
@@ -0,0 +1,18 @@
+import { compareAsc, compareDesc } from 'date-fns';
+
+type DatesAreEqualType = {
+	dates: Date[];
+	order: 'asc' | 'desc';
+};
+
+const sortDates = ({ dates, order }: DatesAreEqualType) => {
```

Comment:
The same proposal as in `datesAreEqual`, maybe we can use something less redundant?

Reply:
ya that's fine with me.

Reply:
Renamed to just `sort`.
I am thinking if there might be any cases when we have sort dates and tickets at the same time. But as of now, I think that this kind of function is a bit low-level and even if there will be some mixture between dates and tickets we will use some services or other high-level ways.

Reply:
ya i totally agree. tickets and datetimes don't share the same property names for their date fields, so they are going to require different predicates anyways. That said, it's possible that we may see some duplication within those functions and end up extracting some common logic to its own file, but we don't need to be concerned with that now